docs/topics/gradle/gradle-backend-tutorial.md

```diff
@@ -0,0 +1,115 @@
+[//]: # (title: Get started with Gradle and Kotlin/JVM)
+
+This tutorial demonstrates how to use IntelliJ IDEA and Gradle for creating a console application.
+
+To get started, first download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
+
+## Create a project
+
+1. In IntelliJ IDEA, select **File** | **New** | **Project**.
+2. In the panel on the left, select **New Project**.
+3. Name the new project and change its location, if necessary.
+
+   > Select the **Create Git repository** checkbox to place the new project under version control. You will be able to do
+   > it later at any time.
+   >
+   {type="tip"}
+
+4. From the **Language** list, select **Kotlin**.
+
+   ![Create a console application](jvm-new-gradle-project.png){width=700}
+
+5. Select the **Gradle** build system.
+6. From the **JDK list**, select the [JDK](https://www.oracle.com/java/technologies/downloads/) that you want to use in
+   your project.
+    * If the JDK is installed on your computer, but not defined in the IDE, select **Add JDK** and specify the path to the
+      JDK home directory.
+    * If you don't have the necessary JDK on your computer, select **Download JDK**.
+
+7. From the **Gradle DSL** list, select **Kotlin**.
+8. Select the **Add sample code** checkbox to create a file with a sample `"Hello World!"` application.
+9. Click **Create**.
+
+You have successfully created a project with Gradle.
+
+## Explore the build script
+
+Open the `build.gradle.kts` file. This is the Gradle Kotlin build script, which contains Kotlin-related artifacts and other parts required for the application:
+
+```kotlin
+ // For `KotlinCompile` task below
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "%kotlinVersion%" // Kotlin version to use
+    application // Application plugin. Also see 1️⃣ below the code
+}
+
+group = "org.example" // A company name, for example, `org.jetbrains`
+version = "1.0-SNAPSHOT" // Version to assign to the built artifact
+
+repositories { // Sources of dependencies. See 2️⃣
+    mavenCentral() // Maven Central Repository. See 3️⃣
+}
+
+dependencies { // All the libraries you want to use. See 4️⃣
+   // Copy dependencies' names after you find them in a repository
+   testImplementation(kotlin("test")) // The Kotlin test library
+}
+
+tasks.test { // See 5️⃣
+    useJUnitPlatform() // JUnitPlatform for tests. See 6️⃣
+}
+
+tasks.withType<KotlinCompile> { // Settings for `KotlinCompile` tasks
+    // Kotlin compiler options
+    kotlinOptions.jvmTarget = "1.8" // Target version of generated JVM bytecode
+}
+
+application {
+    mainClass.set("MainKt") // The main class of the application
+}
+```
+
+* 1️⃣ [Application plugin](https://docs.gradle.org/current/userguide/application_plugin.html) to add support for building CLI application in Java.
```

Comment:
Well, I found using these emoticons... OK, but nothing special.
What do you think, do you like them?

We could think about them later if we are interested to merge it asap.

Reply:
I like that they are bright and catch some attention. But we can use more traditional ways to do footnotes.